_includes/use-cases/use-case-transport-accept-relationship-change.md

```diff
@@ -0,0 +1,13 @@
+Accepts a relationship change with the given change Id.
+The given change has to belong to the relationship specified in the path.
+
+{% include rapidoc api_route_regex="^put /api/v2/Relationships/{id}/Changes/{changeId}/Accept$" %}
+
+```shell
```

Comment:
Why do we have a shell command here?

Reply:
This is a relic of the first iteration of documentation they are removed and replaced in the other branch.